edx_exams/apps/core/api.py

```diff
@@ -142,6 +143,23 @@ def update_attempt_status(attempt_id, to_status):
     return attempt_id
 
 
+def reset_exam_attempt(attempt, requesting_user):
+    """
+    Reset an exam attempt
+    """
+    course_key = CourseKey.from_string(attempt.exam.course_id)
+    usage_key = UsageKey.from_string(attempt.exam.content_id)
+
```

Comment:
Should we add a log here, similar to [the one we have in edx-proctoring](https://github.com/openedx/edx-proctoring/blob/master/edx_proctoring/api.py#L1906C2-L1906C2)?

Reply:
I think we have a few things to track this that we didn't have when that was written. There's a distinct API request for deletes and we also keep track of changes with the simple history library.